src/server/index.jsx

```diff
@@ -159,6 +162,10 @@ server
           });
         }
 
+        data.remoteConfig = remoteConfig;
+        data.path = urlPath;
+        data.timeOnServer = Date.now();
```

Comment:
As above any reason this has been added in this PR?

Reply:
`path` and `timeOnServer` were added by media team recently, while resolving a conflict I moved them here

Reply:
Actually this was a duplicate issue from a merge conflict. Good spot, fixed now